projects/1.16/assets/growthcraft-community-edition/growthcraft_milk/lang/zh_cn.json

```
@@ -0,0 +1,100 @@
+{
+   "block.growthcraft_milk.appenzeller_cheese": "阿本塞尔奶酪",
+   "block.growthcraft_milk.appenzeller_cheese_curds": "阿本塞尔奶酪凝乳",
+   "block.growthcraft_milk.asiago_cheese": "阿齐亚戈奶酪",
+   "block.growthcraft_milk.asiago_cheese_curds": "阿齐亚戈奶酪凝乳",

```

```suggestion
   "block.growthcraft_milk.asiago_cheese": "艾斯阿格奶酪",
   "block.growthcraft_milk.asiago_cheese_curds": "艾斯阿格奶酪凝乳",
```
这地名叫`阿夏戈`，搜奶酪名叫`艾斯阿格`（）